camelot/core/files/storage.py

```diff
@@ -147,21 +147,21 @@ def checkin(self, local_path: PurePosixPath, filename: os.PathLike = None) -> St
         """
         self.available()
 
-        assert isinstance(local_path, PurePosixPath)
+        assert isinstance(local_path, PurePath)
 
         if filename is None and len(local_path.name) > 100:
             raise UserException(text=ugettext('The filename of the selected file is too long'),
                                 resolution=ugettext('Please rename the file'))
 
-        name = PurePosixPath(filename or local_path)
+        name = PurePath(filename or local_path)
         root, extension = name.stem, name.suffix
 
         handle, to_path = self._create_tempfile_with_user_exceptions(extension, root)
         os.close(handle)
 
         logger.debug(f'copy file from {local_path} to {to_path}')
         shutil.copy(Path(local_path), Path(to_path))
```

Comment:
als local_path al een Path is, dan moet dat hier ook nt meer geconverteerd w

Reply:
Dit was om filename om te zetten en alles op 1 lijn te houden, ik zal het aanpassen